database/configuration/my.cnf

```diff
@@ -11,3 +11,4 @@ init_connect='SET NAMES utf8mb4'
 character-set-server=utf8mb4
 collation-server=utf8mb4_unicode_ci 
 skip-character-set-client-handshake
+lower_case_table_names=1
```

Comment:
Hi,
This is a configuration specific for MacOS developers. Please do not push the change to the repository